data_validation/config_manager.py

```diff
@@ -712,19 +743,40 @@ def build_config_calculated_fields(
 
         return calculated_config
 
+    @staticmethod
+    def _is_oracle_client(client):
```

Comment:
A max size per database could live in clients.py?  Something like
```
def get_max_column_length(client):
  if ..
  else:
      return 128
```

Reply:
Thanks, that's a good idea. I initially wanted to add a method to each client but I was nervous of making such a broad change so I added in `config_manager.py`. I'll move the logic to `client.py`.

Reply:
I've made that change.